{"name": "测试数据：【 wintest3，False，3006，该用户不允许删除 】", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'get'", "trace": "self = <testcases.api_test.test_05_delete_user.TestUserDelete object at 0x10a1d7c10>\nlogin_fixture = {'code': 1003, 'msg': '用户名不存在！！！'}, username = 'wintest3'\nexcept_result = False, except_code = 3006, except_msg = '该用户不允许删除'\n\n    @allure.story(\"用例--删除用户信息\")\n    @allure.description(\"该用例是针对获取用户删除接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{except_result}，{except_code}，{except_msg} 】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, except_result, except_code, except_msg\",\n                             api_data[\"test_delete_user\"])\n    @pytest.mark.usefixtures(\"insert_delete_user\")\n    def test_delete_user(self, login_fixture, username, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        user_info = login_fixture\n>       admin_user = user_info.get(\"login_info\").get(\"username\")\nE       AttributeError: 'NoneType' object has no attribute 'get'\n\ntestcases/api_test/test_05_delete_user.py:36: AttributeError"}, "description": "该用例是针对获取用户删除接口的测试", "attachments": [{"name": "log", "source": "a3a18107-235f-44fa-875a-0f0736ed2000-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "7b736ef9-f4cd-4970-b88d-e65d71ab78ef-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'wintest3'"}, {"name": "except_result", "value": "False"}, {"name": "except_code", "value": "3006"}, {"name": "except_msg", "value": "'该用户不允许删除'"}], "start": 1744699123548, "stop": 1744699123549, "uuid": "9b49a293-2f38-43cc-bfaf-209ef665e9fc", "historyId": "6a8136563fec3910840f1add5a7a645c", "testCaseId": "f140003f8969cec64326a77ad093297e", "fullName": "testcases.api_test.test_05_delete_user.TestUserDelete#test_delete_user", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--删除用户信息"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "feature", "value": "用户删除模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_05_delete_user"}, {"name": "subSuite", "value": "TestUserDelete"}, {"name": "host", "value": "192.168.10.5"}, {"name": "thread", "value": "23339-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_05_delete_user"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}